### ISSUE TITLE
propertyOrdering and singleLinePerProperty bugs

### ISSUE THREAD

**Issue by wareczek** [lesshint/lesshint#177]
```
.form-group {
    margin-bottom: 0;

    .form-control {
        height: auto;
    }
}
```

> Warning: bootstrap.less: line 5, col 9, propertyOrdering: Property ordering is not alphabetized
> Warning: bootstrap.less: line 5, col 9, singleLinePerProperty: Each property should be on its own line.

```
.form-group {
    margin-bottom: 0;
}

.form-control {
    height: auto;
}
```

> Warning: bootstrap.less: line 2, col 5, singleLinePerProperty: Each property should be on its own line.
> Warning: bootstrap.less: line 6, col 5, singleLinePerProperty: Each property should be on its own line.

**Comment by shellscape** [comment]
what version are you using?

**Comment by wareczek** [comment]
As in title ;) 2.0 rc

**Comment by jwilsson** [comment]
I got a fix for the `propertyOrdering` issue in [bug-2.0/propertyOrdering](https://github.com/lesshint/lesshint/commits/bug-2.0/propertyOrdering).

**Comment by wareczek** [comment]
fix for `singleLinePerProperty` bug:

`if (!next || (next && ['comment', 'rule'].indexOf(next.type))) {`

on line:
https://github.com/lesshint/lesshint/blob/master/lib/linters/single_line_per_property.js#L6

@jwilsson correct? ;) sorry for no PR, i'm busy now :(

**Comment by jwilsson** [comment]
Fully closed with 943f7c1.
